sql/core/src/test/scala/org/apache/spark/sql/analysis/AnalysisConfOverrideSuite.scala

```diff
@@ -47,7 +48,7 @@ class AnalysisConfOverrideSuite extends SharedSparkSession {
       val key = "spark.sql.catalog.x.y"
       val value = "true"
       withSQLConf(key -> value) {
-        f
+        f(key, value)
```

Comment:
`f` is not a thunk, thus all test bodies were not executed before.

Reply:
!!!! Thanks for the fix @pan3793